pkg/relayer/proxy/metrics.go

```diff
@@ -0,0 +1,90 @@
+package proxy
+
+import (
+	stdprometheus "github.com/prometheus/client_golang/prometheus"
+
+	"github.com/go-kit/kit/metrics"
+	"github.com/go-kit/kit/metrics/prometheus"
+)
+
+var (
+	// relaysTotal is a counter metric representing the total number of requests processed by the relay miner.
```

Comment:
DItto my comment in the other `mtrics.go` file.

1. Try to simplify comments a bit (put yourself in the shoes someone reading this)
2. Wanted to discuss adding a pre/post fix to these.